lib/tasks/sul.rake

```diff
@@ -197,7 +197,7 @@ namespace :sul do
           puts "#{message} : #{contributions.size} publications"
           contributions.each do |contribution|
             pub = contribution.publication
-            next unless !pub_years.empty? && pub_years.include?(pub.year)
+            next unless pub_years.empty? || (pub_years.present? && pub_years.include?(pub.year))
 
```

Comment:
if the `pub_years` array was empty, the current logic just skips every publication, which is not correct